frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -2136,7 +2293,8 @@ end
 function ReaderHighlight:getSavedExtendedHighlightPage(highlight, page, index)
     local item = {
         datetime = highlight.datetime,
-        drawer   = highlight.drawer,
+        drawer   = highlight.drawer or self.highlight.saved_drawer,
```

Comment:
It should inherit drawer from the parent.

Reply:
It does?

I don't rightly recall the reasoning behind the fallback, but that was a suggestion you made in https://github.com/koreader/koreader/pull/11044/files#diff-d5c59944a61752979b90d222e178b78cf71e4b29d8a6714a6c7a6d6decc21204

Reply:
Okay, not exactly the same place apparently, but the code quoted doesn't really exist anymore, so I'm even more confused ;).

Please offer suggestions ;).

Reply:
I think that `item.drawer` always exists for highlights, no need for fallback.

Reply:
And I guess we still need the color fallback for older highlights, though, right?